src/materials/Material.js

```diff
@@ -362,9 +381,10 @@ Material.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 		this.depthWrite = source.depthWrite;
 
 		this.stencilWrite = source.stencilWrite;
+		this.stencilWriteMask = source.stencilWriteMask;
 		this.stencilFunc = source.stencilFunc;
 		this.stencilRef = source.stencilRef;
-		this.stencilMask = source.stencilMask;
+		this.stencilFuncMask = source.stencilFuncMask;
```

Comment:
It might also be a good idea to enhance `MaterialLoader` so these properties can be actually deserialized.